src/sdk.js

```diff
@@ -35,7 +34,7 @@ class Smartcar {
     this.redirectUri = options.redirectUri;
     this.scope = options.scope;
     this.onComplete = options.onComplete;
-    this.development = options.development || false;
+    this.testMode = options.testMode || false;
```

Comment:
not necessary to do, but could refactor this to do the ternary here rather than in the interpolation below:

```js
this.mode  = options.testMode ? 'test' : 'live';
```

same with the scope:

```js
this.scope = encodeURIComponent(options.scope.join(' '));
```

Reply:
I actually did it that way locally at first, but then changed it to what it is now because the other SDKs did it that way. I'm assuming the reasoning is so that if you do `instance.testMode` it's the same as what you passed in? Not really sure, just deferred to consistency here

Reply:
Hmm I see, I guess it depends of you can consider the `this` props to be public or private. Do we document the props in the JS doc?

Reply:
lmao to me they fall on this weird boundary between public and private, but technically private. No we do not